azkaban-web-server/src/main/java/cloudflow/tables.sql

```diff
@@ -20,4 +20,15 @@ CREATE TABLE IF NOT EXISTS `space_watcher` (
                                                `space_id` int(11) NOT NULL,
                                                `username` varchar(45) NOT NULL,
                                                PRIMARY KEY (`space_id`,`username`)
-);
\ No newline at end of file
+);
+
+ALTER TABLE projects
+ADD COLUMN space_id INT NOT NULL,
+ADD COLUMN created_by VARCHAR(45) NOT NULL;
```

Comment:
Perhaps it's safer without the `NOT NULL` requirements so that underlying Azkaban features and existent unit tests still work.